Presentation.Web/Tests/data-processing/tabs/data-processing-registration.oversight.e2e.spec.ts

```diff
@@ -36,29 +36,48 @@ describe("Data processing registration oversight detail tests", () => {
 
     it("Is able to set oversight data",
         () => {
-            var name = createName();
-            var intervalRemark = createRemark();
+            const name = createName();
+            const intervalRemark = createRemark();
+            const completedRemark = createRemark();
             const dropdownInterval = "Hver andet år";
+            const dropdownCompleted = "Ja";
+            const oversightIntervalId = "s2id_oversightInterval_config";
+            const oversightCompletedId = "s2id_oversightCompleted_config";
+            const date = DateHelper.getTodayAsString();
 
+            // Creating and navigating to DPR
             dpaHelper.createAndOpenDataProcessingRegistration(name)
                 .then(() => pageObject.getOversightPage())
+            // Inserting data into completed oversight section
+                .then(() => expectOversightCompletedLatestDateVisibility(false))
+                .then(() => dpaHelper.changeOversightCompleted(dropdownCompleted))
+                .then(() => dpaHelper.changeOversightCompletedLatestDate(date))
+                .then(() => pageObject.getOversightCompletedRemark().sendKeys(completedRemark))
+            // Inserting data into interval oversight section
                 .then(() => pageObject.getOversightIntervalOptionRemark().sendKeys(intervalRemark))
                 .then(() => dpaHelper.changeOversightInterval(dropdownInterval))
-                .then(() => verifyOversightInterval(dropdownInterval))
-                .then(() => verifyOversightIntervalRemark(intervalRemark));
+                // verifying fields for interval oversight section
+                .then(() => verifySelect2Value(dropdownInterval, oversightIntervalId))
+                .then(() => VerifyAttributeValueIs(intervalRemark, pageObject.getOversightIntervalOptionRemark()))
+            // verifying fields for completed oversight section
+                .then(() => VerifyAttributeValueIs(completedRemark, pageObject.getOversightCompletedRemark()))
+                .then(() => verifySelect2Value(dropdownCompleted, oversightCompletedId ))
+                .then(() => expectOversightCompletedLatestDateVisibility(true))
+                .then(() => VerifyAttributeValueIs(date, pageObject.getLatestOversightCompletedDate()));
         });
 
-    function verifyOversightInterval(selectedValue: string) {
-        console.log(`Expecting oversight interval to be set to: ${selectedValue}`);
-        expect(Select2Helper.getData("s2id_oversightInterval_config").getText()).toEqual(selectedValue);
+    function verifySelect2Value(selectedValue: string, selectedId: string) {
```

Comment:
Lav hellere en verify for hver af dem. Så kan du også hive dine consts ned i metode i stedet for at de står i testen